### ISSUE TITLE
Show date changes in tabs + log files

### ISSUE THREAD

**Issue by osa1** [osa1/tiny#304]
When running tiny on a server for long periods of time it would be useful if we showed date changes in the tabs + log files. Currently it's a bit confusing as a timestamp can appear multiple times in the same tab and log file when tiny runs for many days.

cc @Kabouik

**Comment by 097115** [comment]
@osa1 

Or, may be, allow a user to have a configurable timestamp? Some would have nothing, others would settle with `%H:%M`, while some would prefer `%Y-%m-%d %H:%M:%S`...

Our displays are quite wide today, so why not allow users to have this information if they really want it? :)

**Comment by osa1** [comment]
@097115 

> Our displays are quite wide today, so why not allow users to have this information if they really want it? :)

I agree :-) This feature was requested a few times before and we have an issue for it already: https://github.com/osa1/tiny/issues/159.

Unfortunately I don't have a lot of time to work on tiny these days, so this feature needs to be implemented by someone else. I'm happy to help with the implementation and review and merge any PRs related to this.

**Comment by 097115** [comment]
(Just wanted to say that you are really one of the nicest devs I've ever met. Definitely would like to help but unfortunately not a programmer myself :)
